tortoise/tests/test_two_databases.py

```diff
@@ -0,0 +1,60 @@
+from sqlite3 import OperationalError
+
+from tortoise import Tortoise
+from tortoise.contrib.test import TestCase
+from tortoise.tests.testmodels import EventTwo, TeamTwo, Tournament
+from tortoise.utils import generate_schema
+
+
+class TestTwoDatabases(TestCase):
+    async def setUp(self):
+        await self._tearDownDB()
+        self.db = await self.getDB()
+        self.second_db = await self.getDB()
```

Comment:
Here is problem for postgres - you asking for creating of two dbs, but you will get just one - because currently postgres doesn't randomize any parameters and so test fails on dbdelete because it is trying to delete db, while `second_db` still using it